global.json

```diff
@@ -1,8 +1,3 @@
 ﻿{
-    "sources": [ "src", "test" ],
-    "sdk": {
-            "architecture": "x86",
-      "runtime": "clr",
```

Comment:
Seems like CLI is creating WIN7-x64 build configuration by default (verified it with a sample application). Please let me know if it can be configured from here.